### PR TITLE
Update `F.multi_label_soft_margin_loss/nn.MultiLabelSoftMarginLoss` docs 易用性提升

### PR DESCRIPTION
**Pull request by NKNaN**
### PR Category

User Experience

### PR Types

Improvements

### Description

原文档公式是MultiLabelMarginLoss的公式 (参考 https://pytorch.org/docs/stable/generated/torch.nn.MultiLabelMarginLoss.html)
改为MultiLabelSoftMarginLoss对应公式 (参考https://pytorch.org/docs/stable/generated/torch.nn.MultiLabelSoftMarginLoss.html)

### CODE REVIEW

File: python/paddle/nn/functional/loss.py [hunk 1]
```diff
@@ -3449,13 +3449,11 @@ def multi_label_soft_margin_loss(
     For each sample in the mini-batch:
 
     .. math::
-        \text{loss}(x, y) = \sum_{ij}\frac{\max(0, 1 - (x[y[j]] - x[i]))}{\text{x.size}(0)}
+        \text{loss}(x, y) = - \frac{1}{C} * \sum_i y[i] * \log((1 + \exp(-x[i]))^{-1})
```
Comment (zhwesky2010) [inline python/paddle/nn/functional/loss.py:3452]: 是文档写成了`MultiLabelMarginLoss`，但是实现是`MultiLabelSoftMarginLoss ` 对吧？`MultiLabelSoftMarginLoss ` 没有问题吧

Comment (NKNaN) [reply python/paddle/nn/functional/loss.py:3452]: 是的，实现是MultiLabelSoftMarginLoss的实现

**Review by zhwesky2010 — APPROVED** [review]
LGTM
